packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -6361,3 +6861,214 @@ Property details
+| authScheme              | defines how to  connection is authenticated     | `TOKEN`, `UNKNOWN`                                                           | FALSE    | FALSE  | FALSE    |           |
+| _links            | discoverable resources related to the connection            | [JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06)              | TRUE    | FALSE   | TRUE    |           |
+| status            | status of the connection      | `ENABLED`, `DISABLED`, `UNKNOWN`  | FALSE    | FALSE   | TRUE    | `DISABLED` |
+
+An `UNKNOWN` `authScheme` signifies that the authentiication scheme used by the application is not supported yet or that the application does not support provisioning. An `UNKNOWN` `authScheme` will result in an `UNKNOWN` `status`.
+
+### Provisioning Connection Profile object
+
+Defines the application provisioning connection profile. Currently, only token based profile connection is supported.
+
+#### Token based Provisioning Connection Profile example
+```json
+{
+    "profile": {
+        "authScheme": "TOKEN",
+        "token": "TEST"
+    }
+}
+```
+
+#### Token based Provisioning Connection Profile properties
+
+| Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly |
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- | ---------- |
+| authScheme              | defines how to  connection is authenticated     | `TOKEN` | FALSE    | FALSE  | FALSE    |
+| token            | token used to authenticate with application      | String | FALSE    | TRUE   | FALSE    |
+
+### Application Feature object
+
+Defines the application feature.
+
+#### Application Feature example
+```json
+{
+    "name": "USER_PROVISIONING",
+    "status": "ENABLED",
+    "description": "User provisioning settings from Okta to a downstream application",
+    "capabilities": {
+        "create": {
+            "lifecycleCreate": {
+                "status": "DISABLED"
+            }
+        },
+        "update": {
+            "profile": {
+                "status": "DISABLED"
+            },
+            "lifecycleDeactivate": {
+                "status": "DISABLED"
+            },
+            "password": {
+                "status": "DISABLED",
+                "seed": "RANDOM",
+                "change": "KEEP_EXISTING"
+            }
+        }
+    },
+    "_links": {
+        "self": {
+            "href": "http://rain.okta1.com:1802/api/v1/apps/0oa1h43s0sZyK6Ft40g4/features/USER_PROVISIONING",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        }
+    }
+}
+```
+
+#### Application Feature properties
```

Comment:
No header for the properties table, capitals on all the descriptions

Reply:
see: https://github.com/okta/okta-developer-docs/pull/1675/files#r543633227